### DISCUSSION TITLE
Webinar: Tuesday, November 15, 2022 3:30pm – 4:30pm ET

### DISCUSSION THREAD

**Discussion by shaselton-usds** [CMSgov/price-transparency-guide#628] [Announcements]
Webinar: Compliance with Transparency in Coverage Final Rules’ Machine-Readable File Requirements

On Tuesday, November 15, the Centers for Medicare & Medicaid Services (CMS) will host the sixth in a series of technical-guidance webinars related to implementation of the in-network negotiated rate and out-of-network historical billed charges machine-readable files (MRF) requirements in the Transparency in Coverage (TiC) Final Rules (85 FR 72158).

Beginning July 1 of this year, plans and issuers were required to start posting MRFs monthly. Now that we’re a few months out from the first posting,  CMS wants to remind plans and issuers of the requirements for how to post the files, focusing on areas where plans and issuers may need additional clarification. We recommend teams invite their legal or policy teams to attend.  As always, we will be sure to reserve time at the end for questions.

Please register for the webinar [here](https://cms.zoomgov.com/webinar/register/WN_YxSCRBo4SkmN0VJWvn0dcw).

Questions for the webinar can be submitted in advance here on this discussion or via our [website](https://www.cms.gov/healthplan-price-transparency/contact-us).

The webinar series will be posted on the [TiC website](https://www.cms.gov/healthplan-price-transparency/resources/webinars).

**Comment by bettinavanover** [comment]
Hello - when will the webinar recording be published to the TiC website?

**Reply by JR-CMS** [reply]
The 11/15/22 webinar recording will not be published. All information discussed was drawn from material available on our website.

**Comment by dgolden** [comment]
Strictly speaking the webinar did not provide any _new_ information, but that assumes you read the GitHub discussion threads with the intensity of a Bible study group. What the webinar did provide was a useful compendium of important guidance drawn from numerous GitHub discussions.  In  my opinion the over-all takeaway was "we can do better" in MRF production. Note that CMS avoided any finger pointing, and there was no mention of fines. 

Note that there was no guidance as to whether the 1/1/2023 shopping list deadline might slip. So best to assume it will not.

In support of our collective goal of price transparency, I've attached a PDF of screenshots I took of the presentation. The usual caveats apply, as I was taking manual screenshots for my personal use. I skipped a few screens  -- hopefully only screens that were incomplete -- as I usually wait until the last bullet point is revealed before doing a capture.

[CMS_2022_11_15_Webinar.pdf](https://github.com/CMSgov/price-transparency-guide/files/10116645/CMS_2022_11_15_Webinar.pdf)
